### PR TITLE
Hw01

### PR DESCRIPTION
**Pull request by Vlad311010**

### CODE REVIEW

File: hw01/t1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+﻿using System.Text;
+
+namespace t1
+{
+    internal class Program
+    {
+        private static void ListAADigits(int a, int b)
+        {
```
Comment (IgorKheidorov) [inline hw01/t1/Program.cs:8]: Not clear, comment is needed. I understand that 130 is minimum value with AA, by this is not fully truth )) a nd be can be negative

File: hw01/t1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            if (b < 130)
+                return;
+
+            StringBuilder stringBuilder = new StringBuilder();
+            for (int i = a; i <= b; i++)
+            {
```
Comment (IgorKheidorov) [inline hw01/t1/Program.cs:14]: decimalNumber  can be negative, but this case is not proceeded

File: hw01/t1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+namespace t1
+{
+    internal class Program
+    {
+        private static void ListAADigits(int a, int b)
+        {
+            if (b < 130)
+                return;
+
```
Comment (IgorKheidorov) [inline hw01/t1/Program.cs:11]: a can be greater then b

File: hw01/t1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                        case 11:
+                            stringBuilder.Insert(0, 'B');
+                            break;
+                        default:
+                            stringBuilder.Insert(0, remainder);
+                            break;
+                    }
+                }
+
+                if (aCounter == 2)
```
Comment (IgorKheidorov) [inline hw01/t1/Program.cs:36]: Then the application displays all integers in the range from a (inclusive) to b (inclusive), it's supposed to output value is decimal form (just Console.WriteLine(decimalNumber))

File: hw01/t1/Program.cs [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+namespace t1
+{
+    internal class Program
+    {
+        private static void ListAADigits(int a, int b)
+        {
+            if (b < 130)
+                return;
+
+            StringBuilder stringBuilder = new StringBuilder();
```
Comment (IgorKheidorov) [inline hw01/t1/Program.cs:12]: for(int decimallNumber = a; decimalNumber <=b; decimalNumber++)

File: hw01/t2/Program.cs [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    internal class Program
+    {
+        private static string CreateISBN(string digits)
+        {
+            int checkSum = 0;
+            for (int i = 0; i < digits.Length; i++)
+            {
+                checkSum += (10 - i) * int.Parse(digits[i].ToString());
+            }
+
```
Comment (IgorKheidorov) [inline hw01/t2/Program.cs:12]: What about  remainder=0 ? (for example, checkSum= 22). Please fix

**Review by IgorKheidorov — CHANGES_REQUESTED** [review]
Totally cool, fix issues and ask for a review again

File: hw01/t2/Program.cs (hunk 6 above)
Comment (Vlad311010) [reply hw01/t2/Program.cs:12]: fixed in [b79bbf7](https://github.com/Vlad311010/homework/pull/1/commits/b79bbf7db295c6fec1711aceb4f46d0a84a1c37e)

File: hw01/t1/Program.cs (hunk 1 above)
Comment (Vlad311010) [reply hw01/t1/Program.cs:9]: fixed in [cdbf4a1](https://github.com/Vlad311010/homework/pull/1/commits/cdbf4a14117a1e467cf8696832f533825c749aa2)

File: hw01/t1/Program.cs (hunk 2 above)
Comment (Vlad311010) [reply hw01/t1/Program.cs:21]: fixed in [cdbf4a1](https://github.com/Vlad311010/homework/pull/1/commits/cdbf4a14117a1e467cf8696832f533825c749aa2)

File: hw01/t1/Program.cs (hunk 3 above)
Comment (Vlad311010) [reply hw01/t1/Program.cs:18]: fixed in [cdbf4a1](https://github.com/Vlad311010/homework/pull/1/commits/cdbf4a14117a1e467cf8696832f533825c749aa2)

File: hw01/t1/Program.cs (hunk 4 above)
Comment (Vlad311010) [reply hw01/t1/Program.cs:46]: fixed in [cdbf4a1](https://github.com/Vlad311010/homework/pull/1/commits/cdbf4a14117a1e467cf8696832f533825c749aa2)

File: hw01/t1/Program.cs (hunk 5 above)
Comment (Vlad311010) [reply hw01/t1/Program.cs:19]: fixed in [cdbf4a1](https://github.com/Vlad311010/homework/pull/1/commits/cdbf4a14117a1e467cf8696832f533825c749aa2)

**Review by IgorKheidorov — APPROVED** [review]

**Pull request merged**